packages/schema/lib/schemas/RefResourceSchema.js

```diff
@@ -7,18 +7,29 @@ module.exports = makeSchema({
   description:
     'Reference a resource by key and the data it returns. In the format of: `{resource_key}.{foreign_key}(.{human_label_key})`.',
```

Comment:
Could also expand these docs to include some of the fancier references (specifically the `[]`). But, only if we want people to use them. If it's just for compatibility, then we can probably ignore.